### PR TITLE
Feature/csvslave

### PR DESCRIPTION
**Pull request by markaren**
Allow .csv files to be passed to `pythonfmu-builder`. This will produce an FMU emitting the values found in the CSV. The FMU does linear interpolation. 

Cavecats: 
 * Only real values are allowed
 * First row must be time
 * Must specify header
* Must use `,` as seperator

### CODE REVIEW

File: pythonfmu/tests/test_csvslave.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        model.doStep(t, dt)
+        t += dt
+
+    model.instantiate()
+    model.setupExperiment()
+    model.enterInitializationMode()
+    model.exitInitializationMode()
+
+    for i in range(1, 6):
+        numpy.allclose(model.getReal(vrs), [1 * i, pow(2, i)])
```
Comment (markaren) [inline pythonfmu/tests/test_csvslave.py]: Is there an alternative way of comparing lists of floats without 3rd part dependencies?

**Comment by fcollonval** [comment]
I'm a bit sceptical about this one. I get the use case. However it looks like an example rather than a feature for the FMU facade as it is not related to the ability of wrapping a Python script within a FMU (i.e. you could have implemented it using any other language).

It may be more appropriate to put it in the `examples` folder.

Regarding the error, it is a consequence of the non-python files not being included when packaging `pythonfmu`. You will need to add the csv file in the `MANIFEST.in` file to have it included in the package.

```
include pythonfmu/tests/csvdemo.csv
```

File: pythonfmu/tests/test_csvslave.py (hunk 1 above)
Comment (fcollonval) [reply pythonfmu/tests/test_csvslave.py]: Not that I now of.

> Note: for testing, `fmpy` and `pyfmi` are installed. So `numpy` got installed as it is a dependency. 

You should use `numpy.testing.assert_allclose` to raise an error. [`numpy.allclose`](https://docs.scipy.org/doc/numpy-1.17.0/reference/generated/numpy.allclose.html) will return `True` or `False`. But it won't raise an error.

**Comment by markaren** [comment]
> I'm a bit sceptical about this one.

Don't be ^^. It's a killer feature, and we have major use for it in our organization. The reason to integrate it into PythonFMU is to make it streamlined and accessible. 

> i.e. you could have implemented it using any other language

Except you can't in practice. I could have added this feature in FMI4j, which would result in faster execution, but it does not have the streamlined installation procedure as PythonFMU has.

**Pull request merged**